prometheus/filter.go

```diff
@@ -113,6 +114,14 @@ type labelPairs struct {
 	Label []*dto.LabelPair `protobuf:"bytes,1,rep,name=label" json:"label,omitempty"`
 }
 
-func (l *labelPairs) Reset()         {}
-func (l *labelPairs) String() string { return proto.CompactTextString(l) }
-func (*labelPairs) ProtoMessage()    {}
+func (l *labelPairs) Reset() {}
+
+func (l *labelPairs) String() string {
```

Comment:
Do we have testing that validates this is correct?

Reply:
Yes, this is taken from IDPE and they have unit tests covering it

Reply:
discussed offline, but looks like this actually changed.

Reply:
@lesam I updated this function to try and replicate the previous behavior